AsyncSwift/Observed/AppData.swift

```diff
@@ -9,8 +9,68 @@ import SwiftUI
 import UIKit
 
 final class AppData: ObservableObject {
-	@Published var currentTab: Tab = .event // Universal Link로 앱진입시 StampView 전환을 위한 변수
-    var scannedSeminarQR = true // Universal Link로 진입시 QR코드 스캔 여부
+    @Published var currentTab: Tab = .event // Universal Link로 앱진입시 StampView 전환을 위한 변수
+    
+    private var currentStamp: Stamp?
+    lazy var doesStampExist: Bool = {
+        KeyChain.shared.getItem(key: currentStamp?.title) != nil
+    }()
+    
+    init(){
+        fetchCurrentStamp()
+    }
+
+    func checkLink(url: URL) -> Bool {
+        // URL Example = https://www.asyncswift.info?tab=ticketing
+        // URL Example = https://www.asyncswift.info?tab=event
+        guard let host = URLComponents(url: url, resolvingAgainstBaseURL: true)?.host else { return false }
```

Comment:
```suggestion
        guard URLComponents(url: url, resolvingAgainstBaseURL: true)?.host != nil else { return false }
```

값이 사용되지 않는 것 같아서 체크만 하면 어떨까요 ?